pkg/profileparser/profileparser.go

```diff
@@ -602,7 +602,16 @@ func ParseRulesAndDo(contentDom *xmlquery.Node, stdParser *referenceParser, pb *
 
 			description := ruleObj.SelectElement("xccdf-1.2:description")
 			rationale := ruleObj.SelectElement("xccdf-1.2:rationale")
-			warnings := utils.GetWarningsForRule(ruleObj)
+			annotations, err := stdParser.parseXmlNode(ruleObj)
```

Comment:
Future improvement - we can name this to be specific to annotations, since the function is only dealing with annotations.

`parseXmlNode` is really generic.